examples/ec2_remote/index.ts

```diff
@@ -42,31 +45,38 @@ const connection: types.input.remote.ConnectionArgs = {
 };
 
 const connectionNoDialRetry: types.input.remote.ConnectionArgs = {
-    host: server.publicIp,
-    user: "ec2-user",
-    privateKey: privateKey,
-    dialErrorLimit: -1,
```

Comment:
Why are all the changes here necessary?  Shouldn’t the `-1` here have accomplished retrying the connection already?

Reply:
I changed the definition of `connectionNoDialRetry` because it was called `connectionNoDialRetry` and it had the effect of granting infinite dial retries. This change isn't necessary to get the test to pass, it just makes more sense to read. This test is also an example.